odk1-src/collect-import-export.rst

```diff
@@ -1,50 +1,43 @@
+QR codes are a type of barcode that can encode a lot of data with low risk of error. These can be used to configure Collect on a fleet of devices. The QR image displayed on the :guilabel:`Configure via QR code` screen contains all of your current General and Admin settings. When this QR code is scanned from the ODK Collect app on another device, the settings from the code are imported.
 
-.. |share| image:: /img/collect-settings/share-icon.*
-             :alt: Share icon for sharing the QR code. 
-             :height: 43 px
-             :width: 43 px
+.. note::
+  QR codes generated by Collect only contain settings with non-default values because of the constraints on the amount of data a QR code can hold.
 
 .. warning:: 
-  Since the QR code may contain the admin and server passwords without encryption, you should be careful about how you share it. It is advised to not send it through an external application but through *bluetooth*, *MMS* or any other such service that doesn't allow the third party to access the data. 
-
-Saving QR code locally
--------------------------
+  Settings QR codes contain the admin and server passwords *in plain text*. To remove them from the code, :gesture:`tap` the warning on the QR code screen.
 
-You can go to :menuselection:`⋮ --> Save settings to disk` to save the QR code.
+Scanning a QR code
+-------------------
 
-.. note::
+:gesture:`Tap` the :guilabel:`Scan code from other device` button to scan an existing QR code and import all of its settings. The settings contained in the QR code will be set and all others will be reset to their defaults.
 
-  Settings are exported and saved so that they can be loaded to other devices. When you save external settings, they are saved to :file:`/sdcard/odk/settings/collect.settings`. You can load external settings from this location of other devices as well.
+Sharing a QR code
+------------------
 
-  At app launch, the settings are loaded from a different location :file:`/sdcard/odk/collect.settings`, which tracks the state of your settings at all times. 
+You can tap on the :fa:`share-alt` icon to share the QR code as an image. When you click on it, it displays a list of applications and services like *whatsapp*, *facebook*, *hangouts*, *bluetooth*, *MMS* which can be used to share the QR code. This is useful when there are several different data collection sites and all devices have to be configured in the same way, in which case the QR code can be shared from one reference device. 
 
-  Settings are saved to and loaded from different locations to avoid deleting the saved settings.
+.. warning::
+  Since the QR code encodes compressed plain text settings and may contain the admin and server passwords, you should be careful about how you share it. For example, if you print out the QR code and tape it on wall, someone could use a standard QR code scanner to get the admin password.
```

Comment:
Definitely good to have this warning twice given they might only read one of the sections 👍